src/main/java/io/confluent/examples/streams/GlobalStoresExample.java

```diff
@@ -0,0 +1,246 @@
+ * <p>
+ * 1) Start Zookeeper, Kafka, and Confluent Schema Registry. Please refer to <a href='http://docs.confluent.io/current/quickstart.html#quickstart'>QuickStart</a>.
+ * <p>
+ * 2) Create the input/intermediate/output topics used by this example.
+ * <pre>
+ * {@code
+ * $ bin/kafka-topics --create --topic order \
+ *                    --zookeeper localhost:2181 --partitions 4 --replication-factor 1
+ * $ bin/kafka-topics --create --topic customer \
+ *                    --zookeeper localhost:2181 --partitions 3 --replication-factor 1
+ * $ bin/kafka-topics --create --topic product \
+ *                    --zookeeper localhost:2181 --partitions 2 --replication-factor 1
+ * $ bin/kafka-topics --create --topic enriched-order \
+ *                    --zookeeper localhost:2181 --partitions 4 --replication-factor 1
+ * }</pre>
+ * Note: The above commands are for the Confluent Platform. For Apache Kafka it should be
+ * `bin/kafka-topics.sh ...`.
+ * <p>
+ * 3) Start this example application either in your IDE or on the command line.
+ * <p>
+ * If via the command line please refer to <a href='https://github.com/confluentinc/kafka-streams-examples#packaging-and-running'>Packaging</a>.
+ * Once packaged you can then run:
+ * <pre>
+ * {@code
+ * $ java -cp target/kafka-streams-examples-6.0.1-standalone.jar io.confluent.examples.streams.GlobalStoresExample
+ * }
+ * </pre>
+ * 4) Write some input data to the source topics (e.g. via {@link GlobalKTablesExampleDriver}). The
+ * already running example application (step 3) will automatically process this input data and write
+ * the results to the output topic.
+ * <pre>
+ * {@code
+ * # Here: Write input data using the example driver. The driver will exit once it has received
+ * # all EnrichedOrders
+ * $ java -cp target/kafka-streams-examples-6.0.1-standalone.jar io.confluent.examples.streams.GlobalKTablesExampleDriver
+ * }
+ * </pre>
+ * <p>
+ * 5) Once you're done with your experiments, you can stop this example via {@code Ctrl-C}. If needed,
+ * also stop the Confluent Schema Registry ({@code Ctrl-C}), then stop the Kafka broker ({@code Ctrl-C}), and
+ * only then stop the ZooKeeper instance ({@code Ctrl-C}).
+ */
+public class GlobalStoresExample {
+
+    static final String ORDER_TOPIC = "order";
+    static final String CUSTOMER_TOPIC = "customer";
+    static final String PRODUCT_TOPIC = "product";
+    static final String CUSTOMER_STORE = "customer-store";
+    static final String PRODUCT_STORE = "product-store";
+    static final String ENRICHED_ORDER_TOPIC = "enriched-order";
+
+    public static void main(final String[] args) {
+        final String bootstrapServers = args.length > 0 ? args[0] : "localhost:9092";
+        final String schemaRegistryUrl = args.length > 1 ? args[1] : "http://localhost:8081";
+        final KafkaStreams
+                streams =
+                createStreams(bootstrapServers, schemaRegistryUrl, "/tmp/kafka-streams-global-tables");
```

Comment:
Shouldn't be `/tmp/kafka-streams-global-store`?

Reply:
Absolutely. Changed.